File: bit_manipulation/0190_reverse_bits/0190_reverse_bits.go
package main

import "fmt"

func reverseBits(num uint32) uint32 {
	var res uint32

	for i := 0; i <= 31; i++ {
		res <<= 1
		res |= num & 1
		num >>= 1
	}
	return res
}

func main() {
	num := uint32(43261596)
	res := reverseBits(num)
	fmt.Println(res)
}

File: bit_manipulation/0190_reverse_bits/0190_reverse_bits.py
# -*- coding: utf-8 -*-

class Solution:
    # @param n, an integer
    # @return an integer
    def reverseBits(self, n):
        res = 0
        for i in range(32):
            bit = n & 1
            n >>= 1
            res <<= 1
            res += bit
        return res

n = 43261596

print(Solution().reverseBits(n))